src/scss/components/_login-button.scss

```diff
@@ -0,0 +1,18 @@
+@use 'sass:math';
+
+@import '~microscope-sass/lib/typography';
+
+@import '../mixins/prefix';
+
+.#{prefix(login-button)} {
+  display: flex;
+  align-items: center;
+
+  margin-bottom: math.div($grid-container-margin-base, 2);
```

Comment:
this now creates extra white space if there are no login button icons at all - I'd only apply this if it's not the last child (`:not(:last-child)`)

This is also encapsulated in the microscope-sass `@include margin(auto)` which uses the correct/relevant variables for this.